### PR TITLE
Clarifications and style for docs on the wrapped asset bridge

### PR DESCRIPTION
**Pull request by timothymcmackin**
Reduce duplication and overkill to make this easier to follow visually. Link to existing info about tokens on another page and put the info about other tokens in tables for easier skimming. Really a lot of this info should be on the bridge page itself.

### CODE REVIEW

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx [hunk 1]
```diff
@@ -81,221 +86,209 @@ The transfer button can show these errors:
...
-It works by maintaining pools on each side of the bridge.
-
-When you bridge tokens, it locks the tokens on the source network, sends a message over LayerZero's software to the target network, and unlocks wrapped tokens on the target network.
-
-For more information, see the repository for the bridge: https://github.com/LayerZero-Labs/wrapped-asset-bridge. -->
-*/}
\ No newline at end of file
+## Token addresses
+
+For the addresses of wrapped tokens on Etherlink, see [Tokens](./tokens).
```
Comment (rohan-tez) [inline docs/building-on-etherlink/bridging-wrapped-assets.mdx:100]: The only thing is not all the tokens on the token page on Etherlink will be bridgeable on this bridge, for example, WXTZ isn't something included.

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx [hunk 2]
```diff
@@ -81,221 +86,209 @@ The transfer button can show these errors:
...
-
-   This error can occur for bridgeable tokens that exist on multiple networks (ETH/WETH, USDT and USDC). Take the following scenario as an example:
-   - Suppose users have sent 100 USDT from Ethereum to Etherlink, such that 100 USDT is now locked in the bridge contract on Ethereum
-   - 100 USDT is also locked in the bridge contract on Avalanche from users that sent USDT from Avalanche to Etherlink
-   - You hold 150 USDT in your wallet on Etherlink
-   - If you try to send this 150 USDT to Avalanche, it will exceed the amount currently locked in the Avalanche USDT bridge contract; therefore, you can only send up to 100 USDT
+   Select a smaller amount to transfer.
 
+   This error can occur for bridgeable tokens that exist on multiple networks (ETH/WETH, USDT and USDC).
+   For example, if the bridge contract on Avalanche has 100 USDT and you try to transfer 150 USDT from Etherlink to Avalanche, the bridge can't complete the transaction because it doesn't have enough USDT on the target network.
```
Comment (rohan-tez) [inline docs/building-on-etherlink/bridging-wrapped-assets.mdx:92]: can we clarify that a user could transfer up to 100 USDT though?

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx [hunk 3]
```diff
@@ -26,38 +26,43 @@ It allows users to transfer several wrapped tokens, including:
...
-- Please make sure you have imported the correct token into your wallet on the destination chain to see it
+1. Enter the number of tokens to bridge out of the source network.
+
+1. Select the target network and token.
+
+   The page shows the amount of tokens that you will receive on the target chain and the fees for the token transfer in the native gas token of the source chain.
+
+   **You must ensure that you have enough tokens on the source chain to pay the fees.**
+
+1. Set the other parameters for the transfer, including the **Gas on destination** and **Slippage tolerance** fields.
```
Comment (rohan-tez) [inline docs/building-on-etherlink/bridging-wrapped-assets.mdx:52]: Capitalize headings, so 'Gas on Destination' and 'Slippage Tolerance'

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx (hunk 1 above)
Comment (timothymcmackin) [reply docs/building-on-etherlink/bridging-wrapped-assets.mdx:100]: Added "Not all Etherlink tokens can be bridged in and out of Etherlink."

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx (hunk 2 above)
Comment (timothymcmackin) [reply docs/building-on-etherlink/bridging-wrapped-assets.mdx]: > For example, if the bridge contract on Avalanche has 100 USDT, you can transfer no more than 100 USDT from Etherlink to Avalanche.

File: docs/building-on-etherlink/bridging-wrapped-assets.mdx (hunk 3 above)
Comment (timothymcmackin) [reply docs/building-on-etherlink/bridging-wrapped-assets.mdx:52]: They are lower case in the UI: 
Screenshot 2024-07-30 at 11 38 01 AM

**Pull request merged**